c/driver/postgresql/copy/postgres_copy_writer_test.cc

```diff
@@ -260,6 +260,37 @@ TEST(PostgresCopyUtilsTest, PostgresCopyWriteDate) {
   }
 }
 
+TEST(PostgresCopyUtilsTest, PostgresCopyWriteTime) {
+  adbc_validation::Handle<struct ArrowSchema> schema;
+  adbc_validation::Handle<struct ArrowArray> array;
+  struct ArrowError na_error;
+
+  const enum ArrowTimeUnit unit = NANOARROW_TIME_UNIT_MICRO;
```

Comment:
hard coded for now, but when we add nano support this will be parametrized (we have the same pattern for timestamp tests)